_projects/gufe.md

```diff
@@ -1,6 +1,7 @@
 ---
 name: gufe
 repo: OpenFreeEnergy/gufe
+docs: https://gufe.readthedocs.io
```

Comment:
If it doesn't already exist, could you open an issue in gufe to move this to our own domain?

Reply:
do we want it to be like gufe.openfree.energy or something else?

Reply:
I think so, my view is that we definitely should be using a consistent domain for all our docs builds - it makes things nicer / more coherent.